modules/apps/transfer/module.go

```diff
@@ -107,6 +107,10 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 1, m.MigrateTraces); err != nil {
 		panic(fmt.Sprintf("failed to migrate transfer app from version 1 to 2: %v", err))
 	}
+
+	if err := cfg.RegisterMigration(types.ModuleName, 2, m.MigrateMetadata); err != nil {
+		panic(fmt.Sprintf("failed to migrate transfer app from version 2 to 3: %v", err))
+	}
```

Comment:
we'll be running some migrations for total escrow denoms maybe we could include denom metadata migrations within the same consensus version bump. cc @crodriguezvega